.ci/assets/httpie/config.json

```diff
@@ -0,0 +1,7 @@
+{
+    "default_options": [
+      "--ignore-stdin",
+      "--pretty=format",
+      "--traceback"
```

Comment:
Does this mean we get a traceback of httpie in certain failures? I doubt that this is helping us much.

Reply:
yes, here you can see an example:
```
Traceback (most recent call last):
  File "/usr/bin/http", line 11, in <module>
    load_entry_point('httpie==0.9.8', 'console_scripts', 'http')()
  File "/usr/lib/python3/dist-packages/httpie/__main__.py", line 11, in main
    sys.exit(main())
  File "/usr/lib/python3/dist-packages/httpie/core.py", line 227, in main
    log_error=log_error,
  File "/usr/lib/python3/dist-packages/httpie/core.py", line 99, in program
    final_response = get_response(args, config_dir=env.config.directory)
  File "/usr/lib/python3/dist-packages/httpie/client.py", line 70, in get_response
    response = requests_session.request(**kwargs)
  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 506, in request
    prep = self.prepare_request(req)
  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 449, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "/usr/lib/python3/dist-packages/requests/models.py", line 305, in prepare
    self.prepare_url(url, params)
  File "/usr/lib/python3/dist-packages/requests/models.py", line 373, in prepare_url
    raise InvalidURL(*e.args)
requests.exceptions.InvalidURL: Failed to parse: pulp:80null
```
https://github.com/fao89/pulp_ansible/runs/1688323257?check_suite_focus=true#step:9:7263

Reply:
And "Failed to parse: pulp:80null" is all the information we ever need, right?
Do we not get it without `--traceback`? then i will shut up.

Reply:
not sure, need to test to verify

Reply:
yes, we get the message:
```
Still waiting...

http: error: InvalidURL: Failed to parse: pulp:80null
Still waiting...
```
https://github.com/fao89/pulp_ansible/runs/1692070485?check_suite_focus=true#step:9:376